### DISCUSSION TITLE
Artifact update problem: invalid Python version

### DISCUSSION THREAD

**Discussion by foucdeg** [renovatebot/renovate#7990] [Request Help] [answered]
In this PR: https://github.com/foucdeg/passe-un-dessin/pull/671 , the Renovate build generating the Poetry lockfile fails.

Poetry is apparently run under Python 3.9.0, while my project expects the most up-to-date Python 3.9.1 (latest, thanks to Renovate).

How can I make renovate generate the Poetry lockfile using the right Python version?

**Comment by rarkins** [comment, marked as answer]
This was caused by a problem in our build process - unfortunately we hadn't built the latest python image in https://github.com/renovatebot/docker-python

I've fixed the problem and python 3.9.1 should be available within a few hours if you retry/rebase the existing PR
